server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/ACVRUpload.java

```diff
@@ -84,14 +87,16 @@ public String endpoint(final Request the_request, final Response the_response) {
       // we need to create a new CVR instance the "right" way, so it persists and also
       // has the right type
       final CastVoteRecord real_acvr = 
-          CastVoteRecord.instance(RecordType.AUDITOR_ENTERED, Instant.now(), 
+          Persistence.get(new CastVoteRecord(RecordType.AUDITOR_ENTERED, Instant.now(), 
                                   acvr.countyID(), acvr.scannerID(), acvr.batchID(), 
                                   acvr.recordID(), acvr.imprintedID(), acvr.ballotType(), 
-                                  acvr.contestInfo());
+                                  acvr.contestInfo()),
+                          CastVoteRecord.class);
       Main.LOGGER.info("Audit CVR parsed and stored as id " + real_acvr.id());
-      Main.LOGGER.info(CastVoteRecord.getMatching(new HashSet<String>(), 
-                                                  RecordType.AUDITOR_ENTERED).size() + 
-                       " audit CVRs in storage");
+      final long count = count();
+      if (count >= 0) {
```

Comment:
I would prefer we used [`Optional<Long>`](https://docs.oracle.com/javase/8/docs/api/java/util/Optional.html) for methods like this.